src/main/java/io/citrine/jcc/core/CitrinationClient.java

```diff
@@ -43,6 +46,38 @@ public PifSearchResult search(final PifQuery pifQuery) throws IOException {
     }
 
     /**
+     * Request predictions from a model.
+     *
+     * @param modelName to make the prediction against
+     * @param inputs    list of materials, as Maps[String, Object], to make predictions on
+     * @return {@link PredictionResult} containing the results
+     * @throws IOException      when there are serialization issues
+     * @throws RuntimeException if a non-200 response is received.
+     */
+    public PredictionResult predict(String modelName, List<Map<String, Object>> inputs) throws IOException {
+        final HttpPost post = new HttpPost(this.host + "/api/csv_to_models/" + modelName + "/predict");
+        post.addHeader("X-API-Key", this.apiKey);
+        post.addHeader("Content-type", "application/json");
+        Map<String, Object> request = new HashMap<String, Object>();
```

Comment:
This would be more consistent with the rest of the package, like PifQuery objects.